stock_picking_late_activity/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock Picking Late Activity",
+    "summary": "Add an scheduled action that creates late picking activities",
+    "version": "12.0.1.0.0",
+    "development_status": "Stable",
```

Comment:
This development status is not correct:
```suggestion
    "development_status": "Production/Stable",
```